src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml

```diff
@@ -1115,6 +1116,37 @@
           </command:commandLine>
         </command:commandLines>
       </command:example>
+    <command:example>
+        <maml:title>-------  Example2 Get the request and response content for each deployment operation if enabled on new deployment  -------</maml:title>
```

Comment:
1. nit: Example 2
2. Description should go to next line to be consistent with Example 1 above